README.Rmd

```diff
@@ -107,21 +107,14 @@ reporting_delay <- estimate_delay(
 If data was not available we could instead make an informed estimate of the likely delay (*this is a synthetic example and not applicable to real world use cases and we have not included uncertainty to decrease runtimes*),
 
 ```{r}
-reporting_delay <- dist_spec(
-  mean = convert_to_logmean(2, 1), sd = convert_to_logsd(2, 1), max = 10,
-  dist = "lognormal"
-)
+reporting_delay <- example_reporting_delay
```

Comment:
It seems a bit pointless to reallocate like this? Why not repurpose the text to say using an example and then plot it.

Reply:
I also think we need to mention `dist_spec` here as well

Reply:
TBH I am in favour of leaving this one as is with addition of a plot (can be a new issue)